### PR TITLE
Add centerVertically option

### LINKED ISSUE #120: Center glyphs vertically
### Feature description

Provide an option to vertically center the glyphs in the generated font.

### Use cases

Given these two svgs:

```svg
<!-- horizontal.svg -->
<svg width="20" height="4" viewBox="0 0 20 4" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M0 2C0 3.10457 0.895431 4 2 4C3.10457 4 4 3.10457 4 2C4 0.89543 3.10457 0 2 0C0.895431 0 0 0.89543 0 2Z" fill="#DDDDDD"/>
<path d="M8 2C8 3.10457 8.89543 4 10 4C11.1046 4 12 3.10457 12 2C12 0.89543 11.1046 0 10 0C8.89543 0 8 0.89543 8 2Z" fill="#DDDDDD"/>
<path d="M16 2C16 3.10457 16.8954 4 18 4C19.1046 4 20 3.10457 20 2C20 0.89543 19.1046 0 18 0C16.8954 0 16 0.89543 16 2Z" fill="#DDDDDD"/>
</svg>
```

```svg
<!-- vertical.svg -->
<svg width="4" height="20" viewBox="0 0 4 20" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M2 0C0.895431 0 0 0.895431 0 2C0 3.10457 0.895431 4 2 4C3.10457 4 4 3.10457 4 2C4 0.895431 3.10457 0 2 0Z" fill="#DDDDDD"/>
<path d="M2 8C0.895431 8 0 8.89543 0 10C0 11.1046 0.895431 12 2 12C3.10457 12 4 11.1046 4 10C4 8.89543 3.10457 8 2 8Z" fill="#DDDDDD"/>
<path d="M2 16C0.895431 16 0 16.8954 0 18C0 19.1046 0.895431 20 2 20C3.10457 20 4 19.1046 4 18C4 16.8954 3.10457 16 2 16Z" fill="#DDDDDD"/>
</svg>

```

If I generate the svg font, the horizontal version of the icon will end up bottom aligned, like so:

image

If I use the `normalize` option, the horizontal icon will be stretched to the height of the vertical icon, which is clearly not ideal:

image

The ideal result would be to have both centered vertically, like so:

image

- [x] I will/have implement the feature

I have opened a PR (https://github.com/nfroidure/svgicons2svgfont/pull/119) that adds this option.

Let me know your thoughts.

Thank you for this library btw!

### PR DESCRIPTION
**Pull request by vinicius0026**
Fixes #120 

### Proposed changes
- add `centerVertically` option to center glyphs vertically in the generated font

### Code quality
- [x] I made some tests for my changes
- [x] I added my name in the
 [contributors](https://docs.npmjs.com/files/package.json#people-fields-author-contributors)
 field of the package.json file

### License
To get your contribution merged, you must check the following.

- [x] I read the project license in the LICENSE file
- [x] I agree with publishing under this project license

### CODE REVIEW

File: src/index.js
Comment (nfroidure) [inline src/index.js:487]: You should probably use the `const bounds = glyphPath.getBounds();` here like for the centerHorizontally option.

If your intent is to indeed have the ability to rely on the declared glyph height instead of its real path bounds, you could add an option like `useDeclaredGlyphSizes: h | v | hv` or something like that to release that behavior for both h/v centering or only one of them.

**Review by nfroidure — CHANGES_REQUESTED** [review]
Lil thing to harmonize the options otherwise LGTM.

File: src/index.js
Comment (vinicius0026) [reply src/index.js]: Updated the PR to use get the height from the path bounds, as suggested.

I might create another PR with the `useDeclaredGlyphSizes` option later this week.

**Review by nfroidure — APPROVED** [review]
LGTM. @NaridaL or @sabberworm do you want to have a look too before I merge the PR?

**Comment by vinicius0026** [comment]
hey there! pinging to see if we can get this merged and released.

@NaridaL @sabberworm 

thanks!

**Comment by pioug** [comment]
@nfroidure Can you consider adding as maintainer to this repo, and give me permissions to the npm package. I want to help with the basic chores (update, release...).

**Comment by nfroidure** [comment]
@pioug ya welcome, I do it now.

**Comment by pioug** [comment]
@vinicius0026 Can you make another PR to fix the test that you added? I get this error:

```js
      + expected - actual

             descent="0" />
           <missing-glyph horiz-adv-x="0" />
           <glyph glyph-name="bottomleft"
             unicode="&#xE001;"
      -      horiz-adv-x="100" d="M20 60L20 80L40 80L40 60z" />
      +      horiz-adv-x="100" d="M20 20L20 40L40 40L40 20z" />
           <glyph glyph-name="center"
             unicode="&#xE002;"
      -      horiz-adv-x="100" d="M40 80L40 100L60 100L60 80z" />
      +      horiz-adv-x="100" d="M40 40L40 60L60 60L60 40z" />
           <glyph glyph-name="topright"
             unicode="&#xE003;"
      -      horiz-adv-x="100" d="M60 120L60 100L80 100L80 120z" />
      +      horiz-adv-x="100" d="M60 80L60 60L80 60L80 80z" />
```

**Comment by pioug** [comment]
@vinicius0026 The calculation seems wrong 🤔 The "center" icon shouldn't be affected at all. This gives better results for me:

```js
 glyphPath.translate(0, (fontHeight - (bounds.maxY - bounds.minY))/2 - bounds.minY)
```

and it is exactly like the calculation for horizontal centering but on the Y-axis.

I'll update it myself this weekend if no replies

**Pull request merged**
